### PR TITLE
{vis}[GCCcore/10.3.0] Graphviz v2.47.2 w/ Python 3.9.5

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in total)
node3129.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/c92f63d7b205e14a8f63d30c6c4c56bf for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in total)
node3518.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/92f8eb9ea7459dab396e7872c71347de for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in total)
select-pika-c6gd-2xlarge-0001 - Linux centos linux 8.3.2011, AArch64, ARM UNKNOWN (graviton2), Python 3.6.8
See https://gist.github.com/aaf7729fb2d26b86626e346309003ccb for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in total)
easybuild2.novalocal - Linux centos linux 8.3.2011, POWER, IBM pSeries (emulated by qemu) (power9le), Python 3.6.8
See https://gist.github.com/6d044ded9a0fa39371ee6677c2b34531 for a full test report.

**Comment by branfosj** [comment]
@boegel There is overlap between this and #13202. I've put a change for `Gdk-Pixbuf` on that one.

**Comment by boegel** [comment]
> @boegel There is overlap between this and #13202. I've put a change for `Gdk-Pixbuf` on that one.

requested change for `Gdk-Pixbuf` made in #13202, we can sync this one with `develop` once #13202 is merged?

**Comment by boegel** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/h/HarfBuzz/HarfBuzz-2.6.7-GCCcore-10.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+easyblock = 'ConfigureMake'
```
Comment (branfosj) [inline easybuild/easyconfigs/h/HarfBuzz/HarfBuzz-2.6.7-GCCcore-10.3.0.eb:1]: This is no longer needed.

**Review by branfosj — CHANGES_REQUESTED** [review]

**Comment by branfosj** [comment]
Test report by @branfosj
**FAILED**
Build succeeded for 3 out of 4 (4 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/1505cdc629cb8723aa4004fbb740b587 for a full test report.

**Comment by boegel** [comment]
@branfosj Looks like your `Qt5` module is broken because `LLVM` is no longer in place?
```
== 2021-06-19 13:03:48,931 easyblock.py:3656 WARNING build failed (first 300 chars): Module command '/usr/share/lmod/lmod/libexec/lmod python load Qt5/5.15.2-GCCcore-10.3.0' failed with exit code 1; stderr: Lmod has detected the following error: The following module(s) are unknown: 
        "LLVM/11.1.0-GCCcore-10.3.0"
```

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
bear-pg0211u29a.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/1f56ca5c93442971de4c0b4c508bda11 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
node2618.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/47e00c1eb1a23ae3751f74c4d32c40eb for a full test report.

**Comment by branfosj** [comment]
> @branfosj Looks like your `Qt5` module is broken because `LLVM` is no longer in place?
> 
> ```
> == 2021-06-19 13:03:48,931 easyblock.py:3656 WARNING build failed (first 300 chars): Module command '/usr/share/lmod/lmod/libexec/lmod python load Qt5/5.15.2-GCCcore-10.3.0' failed with exit code 1; stderr: Lmod has detected the following error: The following module(s) are unknown: 
>         "LLVM/11.1.0-GCCcore-10.3.0"
> ```

I've set off a LLVM rebuild on that system. I'm not sure what I've done.

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/4ea9be920ba2bee565918bc3f15d45ec for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Pull request merged**